File: .net/dojos/dojo1/SecondPractice/BowlingSecondPractice/Frame.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BowlingSecondPractice
{
    public class Frame
    {
        public int FirstBall { get; }
        public int SecondBall { get; }

        public Frame(int firstBall, int secondBall)
        {
            this.FirstBall = firstBall;
            this.SecondBall = secondBall;
        }

        public int Score { get { return FirstBall + SecondBall; } }
    }
}

File: dotnet/dojos/dojo1/JBG/src/main/java/bowling/second/practice/Frame.java
package bowling.second.practice;

public class Frame {
    private int firstBall;
    private int secondBall;

    public Frame(int firstBall, int secondBall) {
        this.firstBall = firstBall;
        this.secondBall = secondBall;
    }

    public int countScore() {
        return firstBall + secondBall;
    }
}
